android-components/components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FxaDeviceConstellation.kt

```diff
@@ -61,60 +58,6 @@ class FxaDeviceConstellation(
         None,
     }
 
-    @Suppress("ComplexMethod")
-    @Throws(FxaPanicException::class)
-    override suspend fun finalizeDevice(
```

Comment:
This wasn't for readability, it's because I think that the code needs to live inside `internalStateSideEffects`.  Regardless, I'll split it out so that we can have smaller PRs.

Reply:
Okay, now I actually think it would be great if we could decouple `FxaClient` from `Fenix` side. I think part of the confusion from working with account stuff in Fenix is the amount of various classes that are doing seemingly the same thing. If we can reduce those, it's a +!